api/PerformanceResourceTiming.json

```
@@ -588,13 +588,21 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/PerformanceResourceTiming/finalResponseHeadersStart",
           "spec_url": "https://w3c.github.io/resource-timing/#dom-performanceresourcetiming-finalresponseheadersstart",
           "support": {
-            "chrome": {
-              "version_added": "133"
-            },
+            "chrome": [
+              {
+                "version_added": "133"
+              },
+              {
+                "alternative_name": "responseStart",
+                "version_added": "115",
+                "version_removed": "132"
```

`version_removed` is _not_ the last version, but the first version where it is no longer observed:

```suggestion
                "version_removed": "133"
```